ADAL/src/request/ADAuthenticationRequest+WebRequest.m

```diff
@@ -80,6 +82,45 @@ - (void)executeRequest:(NSDictionary *)request_data
      }];
 }
 
+- (NSString *)getRefreshTokenForRequest
+{
+    if (![NSString msidIsStringNilOrBlank:_refreshToken])
+    {
+        return _refreshToken;
```

Comment:
I cannot find where this variable would get set. Can you clarify?

Reply:
it is set from - [ADAuthenticationRequest setRefreshToken:]. API "136" and "137". We aren't using either of these methods, but if the refresh token is already set for the request I think it is correct to use it instead of looking in the cache.